src/std/main.ab

```diff
@@ -239,3 +239,39 @@ pub fun get_env_var(var: Text): Text {
 pub fun load_env_file(): Null {
     unsafe $export "\$(xargs < .env)" > /dev/null$
 }
+
+pub fun shell_isset(name: Text): Bool {
+    $[[ ! -z \$\{!name+z} ]]$ failed {
+        return false
+    }
+    return true
+}
+
+pub fun shell_constant_set(name: Text, val: Text): Bool {
```

Comment:
This function should be _failable_ I think. If user types a constant that is not a valid shell variable name then it will return false but will not warn the developer / user.